mne/preprocessing/peak_finder.py

```diff
@@ -97,7 +96,7 @@ def peak_finder(x0, thresh=None, extrema=1, verbose=None):
                 length -= 1
 
         # Preallocate max number of maxima
-        maxPeaks = int(ceil(length / 2.0))
+        maxPeaks = int(np.ceil(length / 2.0))
```

Comment:
No need to convert to `int` since `maxPeaks` is only used in `np.zeros`, which works with integer floats as well.

Reply:
Unfortunately not:
```
>>> import numpy as np
>>> np.zeros(1.)
Traceback (most recent call last):
  File "<ipython-input-2-7226ec7f6b9a>", line 1, in <module>
    np.zeros(1.)

TypeError: 'float' object cannot be interpreted as an integer
```

See also

https://travis-ci.org/mne-tools/mne-python/jobs/370006021#L3548

I would do `max_peaks = (length - 1) // 2 + 1` to avoid `float` entirely.

Reply:
Sorry, my bad. Yes, this saves converting back and forth between data types, but at the cost of readability. You could also write `-(-length // 2)`, which I like a bit better. Otherwise, maybe let's just keep using `int(np.ceil(length / 2.0))`.

Reply:
I think i'd keep it with `int(np.ceil(length / 2.0))` for readability. Unfortunately, the doctests keep failing ... any of you have an idea what's the issue?